cfgov/mega_menu/management/commands/import_mega_menu.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import json
+
+from django.core.management.base import BaseCommand
+
+from mega_menu.models import Menu
+
+
+class Command(BaseCommand):
+    help = 'Import mega menu content as JSON'
```

Comment:
Maybe add that it will overwrite any existing mega menu for the given language?

Reply:
Nice suggestion, @schbetsy. Unfortunately it's tricky to add multiple lines in command help, so we're limited to one line here. I went with "Import (and overwrite) language-specific mega menu content as JSON." -- does that capture this well?

Reply:
Works for me 👍